CONTRIBUTING.md

```diff
@@ -23,6 +23,10 @@ If you decide to fix an issue, please be sure to check the comment thread in cas
 
 If somebody claims an issue but doesn’t follow up for more than a week, it’s fine to take it over but you should still leave a comment.
 
+## Package Manager
```

Comment:
I'm not sure that we need to mention it. What about scoping the change to the new "Install dependencies:" step?

Reply:
I think we have many nested directories with their own package.json files. One might want to install dependencies at a different time based on their use case (in which directory they want to work).
The steps which are given above are very specific to the root directory during creating PR.
Adding "Install dependencies:" step will not cover all other directories.

Reply:
I think that we should encourage to run yarn at the root, it will handle the nested folders.

Reply:
Understood. Should I go ahead and remove the "Package Manager" section and keep only step 4 which I added?

Reply:
Yes, please do. I think that we should keep the document as simple as possible, try to channel people into a workflow of multiple steps, one at the time.